javalin/pom.xml

```diff
@@ -0,0 +1,142 @@
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-kotlin</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.velocity</groupId>
+            <artifactId>velocity-engine-core</artifactId>
+            <version>2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.freemarker</groupId>
+            <artifactId>freemarker</artifactId>
+            <version>2.3.28</version>
+        </dependency>
+        <dependency>
+            <groupId>org.thymeleaf</groupId>
+            <artifactId>thymeleaf</artifactId>
+            <version>3.0.9.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.spullara.mustache.java</groupId>
+            <artifactId>compiler</artifactId>
+            <version>0.9.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jtwig</groupId>
+            <artifactId>jtwig-core</artifactId>
+            <version>5.87.0.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>com.mitchellbosecke</groupId>
+            <artifactId>pebble</artifactId>
+            <version>2.4.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.atlassian.commonmark</groupId>
+            <artifactId>commonmark</artifactId>
+            <version>0.11.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>swagger-ui</artifactId>
+            <version>3.17.6</version>
+        </dependency>
+        <!-- END Optional dependencies -->
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.5.1</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.name}</Bundle-Name>
+                        <Export-Package>
+                            io.javalin,
+                            io.javalin.apibuilder,
+                            io.javalin.cookie,
+                            io.javalin.core,
+                            io.javalin.core.util,
+                            io.javalin.json,
+                            io.javalin.rendering,
+                            io.javalin.security,
+                            io.javalin.staticfiles,
+                            io.javalin.validation,
+                            io.javalin.websocket
+                        </Export-Package>
+                        <Private-Package>
+                            io.javalin.*
```

Comment:
Are we using `Private-Package` because there are certain packages we need to include but don't want to export?

Reply:
When I take this line out, when I end up using this bundle it fails with this error: 
```
Caused by: java.lang.NoClassDefFoundError: io/javalin/Handler
...
Caused by: java.lang.ClassNotFoundException: io.javalin.Handler cannot be found by javalin-example
```